Trappist/src/Promact.Trappist.DomainModel/Models/Test/Test.cs

```diff
@@ -22,7 +22,8 @@ public class Test : BaseModel
         public string WarningMessage { get; set; }
         public decimal CorrectMarks { get; set; }
         public decimal IncorrectMarks { get; set; }
-
+        public string QuestionOrder { get; set; }
```

Comment:
I would suggest taking enum for ordering options then string the same enum can be used for Questions Order and Options Order. Please refer Question type of difficulty level enum properties in questions model.